packages/remix/test/integration/package.json

```diff
@@ -7,11 +7,15 @@
     "start": "remix-serve build"
   },
   "dependencies": {
+    "@fastify/formbody": "^7.4.0",
+    "@mcansh/remix-fastify": "^3.2.2",
```

Comment:
This is [breaking the tests](https://github.com/getsentry/sentry-javascript/actions/runs/8582424948/job/23583763073?pr=11261) on Node 14 / 16.
I'd suggest adding an e2e test application under `dev-packages/e2e-tests/test-applications` for Fastify where we can control the environment better and also can test more extensively, instead of adding it to the integration test matrix.

Reply:
@rustworthy, not at all! Been there 😅

There may be a couple of reasons for this local resolution issue, but the steps I use are as below:

- `yarn build:tarball` the whole monorepo.
- `pnpm store prune` to make sure the latest tarball is published to the test registry.
- `yarn test:e2e` or `yarn test:run <test-app-name>` to run the tests.

I'll pull the branch locally and test, do you mind if I push commits to this branch if needed, to save you time?

Thanks a lot for sticking with this! ❤️

Reply:
Please feel free to push onto this branch directly and thanks for giving a hand here!

Reply:
@rustworthy, I have set up the e2e test application for you, there seem to be failing cases, you can run tests as I mentioned above.

Reply:
@onurtemizkan Thanks again for your help! I should have paid more attention to the right tools management (`volta`) and the proxy-registry trick. Added a trouble-shooting section to `e2e-tests/README.md`  (maybe for myself in the future :sweat_smile: ).

As for the test app, I am now on it and planning to do the following:
- add missing standard front-end tests and also back-end tests and make sure they are passing with express and the existing `wrapExpressCreateRequestHandler`;
- switch to `fastify` with the corresponding `wrapFastifyCreateRequestHandler` and make sure tests are still passing.

Reply:
just an update: in progress

Reply:
@onurtemizkan Please have a look

Reply:
I see the `tsc` is  not happy in remix integration tests - I'll fix this.

Reply:
@onurtemizkan Done.